moosez/predict.py

```diff
@@ -113,23 +113,20 @@ def preprocess(original_image_directory: str, model_name: str):
     temp_folder = os.path.join(original_image_directory, constants.TEMP_FOLDER)
     os.makedirs(temp_folder, exist_ok=True)
     original_image_files = file_utilities.get_files(original_image_directory, '.nii.gz')
-    resampled_image = os.path.join(temp_folder, constants.RESAMPLED_IMAGE_FILE_NAME)
 
     # check if the model is a clinical model or preclinical model
     desired_spacing = constants.CLINICAL_VOXEL_SPACING if model_name.startswith(
         'clin') else constants.PRECLINICAL_VOXEL_SPACING
 
     # Resample the images to the desired isotropic voxel size
-    resampled_image = image_processing.resample(input_image_path=original_image_files[0],
-                                                output_image_path=resampled_image,
-                                                interpolation=constants.INTERPOLATION,
-                                                desired_spacing=desired_spacing)
+    resampled_image, resampled_image_path = image_processing.resample(input_image_path=original_image_files[0],
+                                                                      interpolation=constants.INTERPOLATION,
```

Comment:
The output image path is not given and there is no code that handles it in the sense of creating a path in image_processing.resample?